### DISCUSSION TITLE
Change WIZ light IP address

### DISCUSSION THREAD

**Discussion by DrewGilbey** [sbidy/wiz_light#163] [Q&A]
I will start with I am very new at this. I have wiz lights integrated and able to control them with zwave switches. I made some changes to my internet setup and IP addresses changed. 

My question is, is there a easy way to change the IP address of a already integrated light?

I didn't add the lights in file edited or configuration.yaml. Lights where added Configurations, integrations, + integration. I can't seem to find where the actual files are located.  

Thanks in advance

**Comment by sbidy** [comment]
No, I think that's not possible. You have to add the bulbs via wizard. You can try to change the config database within HASS (it is still a yaml file) but this will be maybe regenerated or loss in case of an update.

To make it easier/dynamic to change the IP you can use the host or DNS names of the bulb. These will not be changed in case of a release of the IP.

**Reply by DrewGilbey** [reply]
Where is the wiz info located in the HACS yaml file? I can't seem to find it.

**Reply by DrewGilbey** [reply]
Problem Solved, I purchased new lights for the ones I couldn't get back into HA. Not what I wanted to do but haven't been able to find any help to where HACS and the WIZ integration puts the data for editing.
